packages/components/src/Chip/components/ChipPrefix/Chip.Prefix.tsx

```diff
@@ -5,15 +5,25 @@ import { Icon } from "@jobber/components/Icon";
 import { useChildComponent } from "../../hooks/index";
 import styles from "../../Chip.css";
 
-export function ChipPrefix({ children }: PropsWithChildren) {
+export function ChipPrefix({ children, showBG }: ChipPrefixProps) {
   const singleChild = useChildComponent(
     children,
     d => d.type === Avatar || d.type === Icon,
   );
 
   return (
-    <span className={classNames(styles.prefix, !singleChild && styles.empty)}>
+    <span
+      className={classNames(
+        styles.prefix,
+        !singleChild && styles.empty,
+        showBG,
+      )}
+    >
       {singleChild}
     </span>
   );
 }
+
+interface ChipPrefixProps extends PropsWithChildren {
+  readonly showBG?: boolean;
```

Comment:
Is `showBG` show background? I know it's internal but I think the prop name could benefit from having the whole word - what do you think?

Reply:
It sure does! I've updated it. That's just years of ingrained habit of not wanting to type `background` over and over it's an awkward combination of letters to type. Not nearly as bad as awkward though, phew. :)